pkgs/development/libraries/mesa/default.nix

```diff
@@ -13,6 +13,7 @@
 , withValgrind ? !stdenv.isDarwin && lib.meta.availableOn stdenv.hostPlatform valgrind-light, valgrind-light
 , enableGalliumNine ? stdenv.isLinux
 , enableOSMesa ? stdenv.isLinux
+, libclc
```

Comment:
Note to self: We probably want to package this as `llvmPackages*.libclc` to keep it up-to-date, etc. (outside the scope of this PR but in case anyone has an opinion on that).

Reply:
i suggested it before, but it seem not to be done yet

https://github.com/NixOS/nixpkgs/issues/89187#issuecomment-636227881
https://github.com/NixOS/nixpkgs/pull/111065#issuecomment-879325651